pkg/bridge/ebtables/ebtables.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2017 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ebtables
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	osexec "os/exec"
+	"strconv"
+	"strings"
+	"syscall"
+
+	utilexec "github.com/containernetworking/plugins/pkg/exec"
+	"github.com/containernetworking/plugins/pkg/utils/sysctl"
+)
+
+const (
+	sysctlBridgeCallIPTables = "net/bridge/bridge-nf-call-iptables"
```

Comment:
Can we also set the v6 iptables?
sysctlBridgeCallIP6Tables = "net/bridge/bridge-nf-call-ip6tables"